accern_xyme/accern_xyme.py

```diff
@@ -107,11 +102,6 @@
     "custom_json",
     "custom_json_to_data",
 }
-EMBEDDING_MODEL_NODE_TYPES = {
```

Comment:
put back -- revert all changes from master which are not intentionally made from this PR